src/components/DataViewer/DataViewer.tsx

```diff
@@ -15,45 +17,70 @@ const StyledModal = styled(Modal)`
     left: 50%;
     transform: translate(-50%, -50%);
     border: 0;
-    border-radius: 5px;
+    border-radius: 2em;
     outline: none;
     padding: 0;
     width: 80%;
     max-height: 80%;
-    background: rgb(255, 255, 255);
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
     box-shadow: rgb(200, 200, 200) 0 2px 10px;
     overflow: hidden;
+    max-width: 1000px;
 `;
-const Header = styled.h3`
+
+const Header = styled.div`
     display: flex;
     justify-content: space-between;
     align-items: center;
-    background-color: #eeeeee;
-    padding: 0.5rem;
-    margin-top: 0;
+    color: ${(props) => props.theme.secondaryForegroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}A0;
+    font-size: 1.5em;
+    font-weight: bolder;
+    padding: 0.75em 1em;
 `;
+
 const Content = styled.div`
-    overflow: scroll;
-    margin: 1.5rem;
-    margin-top: 1rem;
+    overflow: auto;
+    margin: 1rem 1.5rem 1.5rem;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
 `;
+
 const Key = styled.div`
-    color: grey;
-    font-size: small;
+    color: ${(props) => props.theme.foregroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}40;
+    display: inline-block;
+    font-size: medium;
+    border-radius: 0.7em;
+    padding: 0.2em 0.5em;
 `;
+
 const Value = styled.div`
     font-size: medium;
+    padding: 0.2em 0.5em;
 `;
+
 const EachItem = styled.div`
     padding-bottom: 1rem;
 `;
-const CloseButton = styled.button`
+
+const CloseButton = styled.button.attrs({
+    "aria-label": "Close Button",
+})`
     border: 0;
-    background-color: red;
-    color: white;
+    border-radius: 50%;
+    color: grey;
```

Comment:
can this be from a theme?

Reply:
Same for this, it is an issue of VFB-16 and we have left a note on this for the person handling this ticket to refactor it once done

Reply:
Can we set this to something in themes, even if it isn't 'as good', e.g. white/black? Why does it need to be grey?